example/src/test/java/com/lightstep/tracer/shared/ExampleTracerTest.java

```diff
@@ -0,0 +1,25 @@
+package com.lightstep.tracer.shared;
+
+import io.opentracing.Span;
+import org.junit.Test;
+
+public class ExampleTracerTest {
+    @Test
+    public void test() throws Exception {
+        ExampleTracer tracer = new ExampleTracer(new Options.OptionsBuilder()
+                .withAccessToken("LIGHTSTEP_LOCAL_DEV_ACCESS_TOKEN")
+                .withCollectorHost("127.0.0.1")
+                .withCollectorPort(9996)
```

Comment:
to be clear this is the part I'm not sure about - change to public collector host/port?

Reply:
Sure. Should I create a fake access token or something?

Reply:
FYI, I am following the convention of https://github.com/lightstep/lightstep-tracer-go/blob/master/examples/trivial.go#L49, maybe I should change it from a test to a function with a main instead.